specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironmentsDaprResiliencyCircuitBreakerPolicies.json

```diff
@@ -0,0 +1,288 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/daprResiliencyCircuitBreakerPolicies": {
+      "get": {
+        "tags": [
+          "DaprResiliencyCircuitBreakerPolicies"
+        ],
+        "summary": "Get the Dapr Resiliency Circuit Breaker Policies for a managed environment.",
+        "operationId": "DaprResiliencyCircuitBreakerPolicies_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "pattern": "^.*",
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DaprResiliencyCircuitBreakerPoliciesCollection"
```

Comment:
>DaprResiliencyCircuitBreakerPoliciesCollection [](http://example.com/codeflow?start=37&length=46)

Can you refer this object from CommonDefinitions.json instead of defining it again?